python/pyspark/ml/util.py

```diff
@@ -134,13 +134,15 @@ class MLWritable(object):
     .. versionadded:: 2.0.0
     """
 
+    @property
+    @since(2.0)
```

Comment:
We should remove the ```since``` annotations, since these mix-ins are private which will not inherited at all.